jsf/subsystem/src/main/java/org/jboss/as/jsf/logging/JSFLogger.java

```diff
@@ -111,11 +111,11 @@
     DeploymentUnitProcessingException jsfInjectionFailed(String slotName, @Cause Throwable cause);
 
     @LogMessage(level = DEBUG)
-    @Message(id = 17, value = "JSF 1.2 classes detected. Using org.jboss.as.jsf.injection.weld.legacy.WeldApplicationFactoryLegacy.")
+    @Message(id = 17, value = "Faces 1.2 classes detected. Using org.jboss.as.jsf.injection.weld.legacy.WeldApplicationFactoryLegacy.")
```

Comment:
@fjuma what should this log message be changed to?  `Faces 1.2` could be changed back to `JSF 1.2` if that fits.  Or changed to `Jakarta Server Faces 2.3` if that fits/works.

Reply:
This needs to reference `1.2`. There are references to both `Faces 1.2` and `JSF 1.2` in https://jakarta.ee/specifications/faces/3.0/jakarta-faces-3.0.html#a4629 so whichever of these that you think is best is fine.